build/spas.js

```diff
@@ -59,6 +59,16 @@ async function buildSPAs(options) {
         { prefix: "signup", pageTitle: "Sign up", noIndexing: true },
         { prefix: "settings", pageTitle: "Account settings", noIndexing: true },
         { prefix: "plus", pageTitle: "Plus", noIndexing: true },
+        {
+          prefix: "plus/deep-dives/planning-for-browser-support",
+          pageTitle: "Planning for browser support ~ Plus",
+          noIndexing: true,
```

Comment:
Note-to-self; we should cease this practice for deep dives. They ought to be indexed in Google. All of them, but only the title and the article lead. The rest of the prose should be protected by actual business logic and APIs etc.